modules/about-etcd-encryption.adoc

```diff
@@ -21,5 +21,7 @@ When you enable etcd encryption, encryption keys are created. These keys are rot
 
 [NOTE]
 ====
-Keep in mind that etcd encryption only encrypts values, not keys. This means that resource types, namespaces, and object names are unencrypted.
+Keep in mind that etcd encryption only encrypts values, not keys. Resource types, namespaces, and object names are unencrypted.
```

Comment:
I don't think that the "Keep in mind" part is really necessary. This statement works on its own 
```suggestion
Etcd encryption only encrypts values, not keys. Resource types, namespaces, and object names are unencrypted.
```